WMFComponents/Sources/WMFComponents/Components/Article Tabs/WMFArticleTabsView.swift

```diff
@@ -134,7 +134,7 @@ public struct WMFArticleTabsView: View {
                 }
                 .frame(width: geometry.size.width, height: geometry.size.height, alignment: .center)
             }
-            .background(Color(theme.paperBackground))
+            .background(Color(theme.midBackground))
```

Comment:
Just a heads up, this went through some design review rounds 😅 can you make this doesn't regress the feedback fix in https://phabricator.wikimedia.org/T398890#11261803?

> please ensure that the empty state background color of tabs overview is set to #gray100, per the designs. as it currently stands: